### ISSUE TITLE
Update scripts to run over a single ontology with local files

### ISSUE THREAD

**Issue by jamesaoverton** [OBOFoundry/OBO-Dashboard#3]
Required for #2. The current code is designed to run over the full set of published files and registry entries for all OBO projects, but we also need to be able to run for a single ontology's local files: an OWL file and maybe a local copy of their OBO registry YAML/Markdown file. It needs to generate all the dashboard files for a single ontology, especially `dashboard.html`. We want to be able to run this easily from an ODK Makefile.

The simplest thing is to copy some files into place, override the `ONTS` variable, and run `make`. This seems crude. It might be better to modify `dashboard.py` with some optional arguments.

**Comment by jamesaoverton** [comment]
#4 goes a long way to addressing this. Once that PR is merged, we'll play with a simple ODK project such as https://github.com/obophenotype/c-elegans-phenotype-ontology and see how it goes.

**Comment by matentzn** [comment]
I dont want to complicate things, but what would be great if the set of ontologies by the dashboard in OBO foundry mode (non-local mode) could be configured to a set of ontologies. I feel responsible for the metadata and continuous cleaning of ca. 30 ontologies, and I would kinda like to see only those on the dashboard :P Dont mind me if this is to complex, really just a nice to have.

**Comment by jamesaoverton** [comment]
Yes, running on an arbitrary set of ontologies is an important use case that we should be explicit about.

Fortunately it's easy, and already covered in the README: `ONTS="obi go eco" make all`

https://github.com/OBOFoundry/OBO-Dashboard#running-over-multiple-ontologies

**Comment by matentzn** [comment]
Amazing! :) Thanks.
